weaver/sdks/fabric/go-sdk/membershipmanager/membership_manager.go

```diff
@@ -0,0 +1,667 @@
+/*
+Copyright 2020 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package membershipmanager
+
+import (
+	"context"
+	"fmt"
+	"crypto"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	protoV2 "google.golang.org/protobuf/proto"
+
+	"github.com/hyperledger/fabric-protos-go-apiv2/common"
+	mspprotos "github.com/hyperledger/fabric-protos-go-apiv2/msp"
+	"github.com/hyperledger/fabric-admin-sdk/pkg/channel"
+	"github.com/hyperledger/fabric-admin-sdk/pkg/identity"
+	"github.com/hyperledger/fabric-gateway/pkg/client"
+	cidentity "github.com/hyperledger/fabric-gateway/pkg/identity"
+
+	cactiprotos "github.com/hyperledger/cacti/weaver/common/protos-go/v2/common"
+)
+
+
+func CreateLocalMembership(walletPath, userName, connectionProfilePath, channelId, weaverCCId string, mspIds []string) error {
+	membership, err := GetMSPConfigurations(walletPath, userName, connectionProfilePath, channelId, mspIds)
+	if err != nil {
+		return err
+	}
+	membership.SecurityDomain = ""		// We don't need this as the Weaver chaincode will internally use a designated keyword
```

Comment:
Actually the chaincode doesn't set `securityDomain` field in membership, may be if you have time you can add that quickly? The designated keyword is only used as `key` but not used to set the field `securityDomain`.

Reply:
Although it doesn't matter much.

Reply:
@sandeepnRES Please go ahead and commit. I've given you access to my fork.